tap_migrations.json

```diff
@@ -10,7 +10,7 @@
   "avidemux": "homebrew/cask",
   "awsenv": "Luzifer/tools",
   "chromedriver": "homebrew/cask",
-  "cloudfoundry-cli": "pivotal/tap",
+  "cloudfoundry-cli": "cloudfoundry/tap",
```

Comment:
No formula exists with the same name, it seems to be called `cf-cli` in the tap. I'm assuming this won't work unless the names are identical. Should we remove it completely instead ?

Reply:
Or hmm, you just need to add `/cf-cli` at the end I figured out, see the google example further up in the file.

Reply:
Thanks for catching that!

Do you know whether this: https://github.com/cloudfoundry/homebrew-tap/blob/master/formula_renames.json#L2 addresses the same issue?

Also, is there a way I can tell `brew search` to use a branch of homebrew core, so that I can test this behavior for myself?